com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -36,9 +36,11 @@ public function upgrade_1002(){
     return TRUE;
   }
```

Comment:
regarding 1001, it's been there since  Nov 3, 2015 and I don't think there is any existing client or install not get and update since then , but let's keep it to be in the safe side , but please remove 1002 then .